File: borrowing/src/main/java/org/acme/Models/Borrowing.java
package org.acme.Models;

import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoEntity;
import java.time.LocalDate;

public class Borrowing extends ReactivePanacheMongoEntity {

    public String bookId;
    public String userId;
    public LocalDate borrowingDate;
    public LocalDate returnDate;
}

File: Mobile_Api/Models/borrowing/Borrowing.cs
namespace Web_Api.Models.borrowing
{
    public class Borrowing
    {

        public string BookId { get; set; }
        public string UserId { get; set; }
        public DateTime BorrowingDate { get; set; }
        public DateTime ReturnDate { get; set; }
    }
}
